fe/fe-core/src/main/java/com/starrocks/external/iceberg/IcebergUtil.java

```diff
@@ -92,16 +93,18 @@ public static HdfsFileFormat getHdfsFileFormat(FileFormat format) {
      */
     public static TableScan getTableScan(Table table,
                                          Snapshot snapshot,
-                                         List<UnboundPredicate> icebergPredicates,
+                                         List<Expression> icebergPredicates,
                                          boolean refresh) {
         if (refresh) {
             table.refresh();
         }
 
         TableScan tableScan = table.newScan().useSnapshot(snapshot.snapshotId()).includeColumnStats();
-        for (UnboundPredicate predicate : icebergPredicates) {
-            tableScan = tableScan.filter(predicate);
+        Expression filterExpressions = Expressions.alwaysTrue();
+        if (icebergPredicates != null) {
```

Comment:
=> if (!icebergPredicates.isEmpty())
`icebergPredicates` var can not be null when through the preProcessConjuncts().

Reply:
done